impl/src/test/java/org/ehcache/impl/internal/store/tiering/CacheStoreTest.java

```diff
@@ -63,28 +71,36 @@
  */
 public class CacheStoreTest {
 
+  @Mock
+  private CachingTier<Number, CharSequence> numberCachingTier;
+  @Mock
+  private AuthoritativeTier<Number, CharSequence> numberAuthoritativeTier;
+  @Mock
+  private CachingTier<String, String> stringCachingTier;
+  @Mock
+  private AuthoritativeTier<String, String> stringAuthoritativeTier;
```

Comment:
This is actually the recommended way of mocking classes with generic parameters ...